packages/x-date-pickers/src/MobileDateTimePicker/MobileDateTimePicker.tsx

```diff
@@ -41,11 +41,13 @@ const MobileDateTimePicker = React.forwardRef(function MobileDateTimePicker<TDat
     seconds: renderTimeViewClock,
     ...defaultizedProps.viewRenderers,
   };
+  const ampmInClock = defaultizedProps.ampmInClock ?? false;
```

Comment:
This would introduce a regression until we align the `DateTimePickerToolbar` with `TimePickerToolbar` by adding the option to show `am/pm` buttons in it.
Do you think we should temporarily set it to `true` for `Mobile` and `Static` pickers until we fix the issue with the toolbar differences? 🤔

Reply:
Reverted in https://github.com/mui/mui-x/pull/8083/commits/a9b1611d02c85997325ee98bc4238640d0d7e959 to avoid regression.